### PR TITLE
add a new page: Getting Started Tutorials

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

TiDB Cloud supports a new page: Getting Started Tutorials

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ aliases: ['/tidbcloud/beta/tidb-cloud-quickstart']
 
 *Estimated completion time: 20 minutes*
 
-This tutorial guides you through an easy way to get started with your TiDB Cloud.
+This tutorial guides you through an easy way to get started with your TiDB Cloud. You can also navigate to the [**Getting Started Tutorials**](https://tidbcloud.com/console/getting-started) page in the TiDB Cloud console to follow the tutorials step by step.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:12]: If users have not signed up for TiDB Cloud yet, they will be redirected to the signup page. What about providing the Getting Started Tutorials after step 1, sinch all operations in the Getting Started Tutorials needs users to have a cluster first.

**Review by shizn — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-quickstart.md (hunk 1 above)
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:12]:
```suggestion
This tutorial guides you through an easy way to get started with your TiDB Cloud. You can also navigate to the [**Getting Started**](https://tidbcloud.com/console/getting-started) page in the TiDB Cloud console to follow the tutorials step by step.
```

**Pull request merged**